### PR TITLE
Initial work

### PR DESCRIPTION
**Pull request by trentmwillis**
This sets up the initial implementation, test cases, and Travis.

### CODE REVIEW

File: tests/index-test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const path = require('path');
+
+const babel = require('babel-core');
+const CompactReexports = require('../lib');
+
+QUnit.module('htmlbars-inline-precompile', function(hooks) {
```
Comment (rwjblue) [inline tests/index-test.js]: wrong module name here

**Review by rwjblue — COMMENTED** [review]
This seems good to me.  I'm happy for this to land as is, and iterate on the various TODO's and comments below.

* Have you confirmed that the default `ember-cli` generated reexports are handled (e.g. the `app/components/foo-bar.js` file created from `ember g component foo-bar` in an addon)?
* Should we use `define('adsf', define.alias('asdf'))` syntax for now?
* Can you add a TODO or followup issue to add a smoke test that actually evaluates the result and confirms they are the same instance?

File: tests/index-test.js (hunk 1 above)
Comment (trentmwillis) [reply tests/index-test.js]: Hehe, that's what I get for copy-pasta

**Comment by trentmwillis** [comment]
Going with the new `define.alias` syntax and we'll just guard for older `loader.js` usage.

The other issues/todos have been documented in https://github.com/ember-engines/babel-plugin-compact-reexports/issues/2.

**Pull request merged**